pkg/commands/plugin/read.go

```diff
@@ -74,13 +74,8 @@ func cmdRead(c *cli.Context) error { // nolint: gocyclo
 	formatter := formatters.NewReadFormatter(c)
 	for _, read := range resp {
 		err = formatter.Add(&scheme.Read{
-			Type: read.Type,
-			Data: map[string]scheme.ReadData{
-				read.Type: {
-					Value:     read.Value,
-					Timestamp: read.Timestamp,
-				},
-			},
+			// TODO: Add data.
```

Comment:
I think we'd want the read data in here with the PR

Reply:
ah, okay. in that case, I'd say that this shouldn't merge into master, but some staging branch instead. then, once all the work is done for the updates, it can be pushed to master all at once. that way the stuff in master stays stable and working, e.g.

```
dev branches          staging branch         master
--------------        --------------         ------

grpc1.0-server  --->  sdk-1.0-updates  --->  master
subcmd-updates  --->
refactoring     --->
1.0-tests       --->
something-else  --->
foo             --->
```

Reply:
That makes more sense. I am just gonna close this pull request for now and create a staging branch. Also, it is said below that merging is locked because I haven't setup GPG correctly yet. Will need sort that out too.

Reply:
ah, forgot that was set on this repo. this could be useful:
https://help.github.com/articles/signing-commits-with-gpg/

and you can set up autosigning for commits
https://stackoverflow.com/questions/10161198/is-there-a-way-to-autosign-commits-in-git-with-a-gpg-key